src/initramfs-tools/loopback-file-system-support.patch

```diff
@@ -1,3 +1,7 @@
+From: Samuel Angebault <staphylo@arista.com>
+Date: Fri, 5 Jun 2020 14:13:24 -0700
+Subject: Add support for loop root mounting
+
```

Comment:
Why add this header?

Reply:
It does not make sense to add the name to other's patch no matter whatever technical issues.

I just testes with the patch with quilt and it works fine. Even better we have infrastructure support (https://github.com/Azure/sonic-buildimage/blob/master/slave.mk#L384-L389). So you may move patches into `src/initramfs-tools.patch/` directory, and create a `series` file to describe patches order.

Reply:
Sure I can use quilt, though it's still a bit sad to dirty a git tree by applying patches on top since git was invented as a patch management mechanism.
As long as it's not a submodule of sonic-buildimage it won't show in `git status` so I guess it's fine.

Unfortunately the `<package>.patch/` approach does not apply here since the `Makefile` is cloning the source tree. Therefore the sources are not available for patching prior to `make` invocation. I'll just add a `series` file and invoke quilt directly from the `Makefile`.